.editorconfig

```diff
@@ -24,3 +24,7 @@ indent_size = 4
 
 [*.js]
 quote_type = single
+
+[*.less]
+indent_style = space
```

Comment:
Oh, sorry, this wasn't supposed to get part of this commit. My IDE fooled around with the first indentation (first indent was 3 spaces 👀 ) so I tried to trick it with this editorconfig rule.